imagetest/test_suites/oslogin/oslogin_test.go

```diff
@@ -99,7 +111,7 @@ func TestOsLoginDisabled(t *testing.T) {
 		t.Fatalf("cannot read /etc/pam.d/sshd")
 	}
 	contents := string(data)
-	if strings.Contains(contents, "pam_oslogin_login.so") || strings.Contains(contents, "pam_oslogin_admin.so") {
+	if strings.Contains(contents, "pam_oslogin_login.so") {
```

Comment:
No more "pam_oslogin_admin.so"?

Reply:
Nope, it was removed, remember?

Reply:
https://github.com/GoogleCloudPlatform/guest-oslogin/pull/116/commits/39240b36b11a146229355f1fcaa22150d11b011c

Reply:
Ah, right you are. Sounds good.